src/components/welcome/Welcome.jsx

```diff
@@ -0,0 +1,132 @@
+import { Button, Box, Typography } from "@mui/material";
+import React, { useState } from "react";
+import Tour from "reactour";
+import Buttons from "./Buttons";
+import QuestionMarkIcon from "@mui/icons-material/QuestionMark";
+
+const Welcome = () => {
+  const steps = [
```

Comment:
think the `steps` could be defined outside of the component, that way we keep the component with only the definitions it needs.

Reply:
Yes, as discussed on Monday, the tour could change depending on the current page